src/celengine/category.cpp

```diff
@@ -109,11 +113,11 @@ void UserCategory::cleanup()
 UserCategory::CategoryMap UserCategory::m_allcats;
 UserCategory::CategorySet UserCategory::m_roots;
 
-UserCategory *UserCategory::newCategory(const std::string &s, UserCategory *p)
+UserCategory *UserCategory::newCategory(const std::string &s, UserCategory *p, const char *d)
```

Comment:
std::string &domain (and NOT just *d* - d? what does it mean???)

Reply:
Because while earlier, single work I rarely named function parameters and there was nobody to tell me this is bad. Sorry, just forgot.